src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3306,6 +3318,64 @@ private void AddAccessorIfAvailable(ArrayBuilder<Symbol> symbols, MethodSymbol a
             }
         }
 
+        internal override byte? GetLocalNullableContextValue()
+        {
+            byte? value;
+            if (!_flags.TryGetNullableContext(out value))
+            {
+                value = ComputeNullableContextValue();
+                _flags.SetNullableContext(value);
+            }
+            return value;
+        }
+
+        private byte? ComputeNullableContextValue()
+        {
+            var compilation = DeclaringCompilation;
+            if (!compilation.ShouldEmitNullableAttributes(this))
+            {
+                return null;
+            }
+
+            var builder = new MostCommonNullableValueBuilder();
+            var baseType = BaseTypeNoUseSiteDiagnostics;
+            if (!(baseType is null))
+            {
+                builder.AddValue(TypeWithAnnotations.Create(baseType));
+            }
+            foreach (var typeParameter in TypeParameters)
+            {
+                typeParameter.GetCommonNullableValues(compilation, ref builder);
+            }
+            foreach (var pair in GetMembersByName())
+            {
+                foreach (var member in pair.Value)
+                {
+                    member.GetCommonNullableValues(compilation, ref builder);
+                }
+            }
+            return builder.MostCommonValue;
```

Comment:
>return builder.MostCommonValue; [](start = 12, length = 31)

It looks like attributes emitted for local functions and lambdas are not factored in. If this is intentional, it would be good to capture that in a comment.